### PR TITLE
#33618: clarify omitting <tbody>

### LINKED ISSUE #33618: Unclear whether omitting `<tbody>` tags (putting `<tr>` directly under `<table>`) is valid
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody

### What specific section or headline is this issue about?

Technical summary

### What information was incorrect, unhelpful, or incomplete?

The MDN `<tbody>` technical summary says:

> A [`<tbody>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody) element's start tag can be omitted if the first thing inside the [`<tbody>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody) element is a [`<tr>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tr) element, and if the element is not immediately preceded by a [`<tbody>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody), [`<thead>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead), or [`<tfoot>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tfoot) element whose end tag has been omitted. (It can't be omitted if the element is empty.) A [`<tbody>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody) element's end tag can be omitted if the [`<tbody>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody) element is immediately followed by a [`<tbody>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody) or [`<tfoot>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tfoot) element, or if there is no more content in the parent element.

That implies that this markup, which omits the `<tbody>` tag and puts `<tr>` directly under `<table>`, is valid:

```html
<table>
    <tr></tr>
</table>
```

It should be considered valid, even though it really means this:

```html
<table>
    <tbody>
        <tr></tr>
    </tbody>
</table>
```

But in #33505, which is basically the same issue on the `<tr>` page, the reporter @rschristian is adament that the shorter form is *not* valid. I did mention the `<tbody>` page, but nothing came of it, so I am opening an issue specifically for the `<tbody>` page.

### What did you expect to see?

See above.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/tbody`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/tbody/index.md
* Last commit: https://github.com/mdn/content/commit/829db137a01feb14af7beaec178a3ea0118b4777
* Document last modified: 2024-03-25T18:37:02.000Z

</details>

### PR DESCRIPTION
**Pull request by estelle**
hopefully fixes #33618

### CODE REVIEW

**Review by wbamberg — CHANGES_REQUESTED** [review]
I didn't like "validly" and thought "may not work as expected or at all" was understating things. Also, we have an example of this in the page!

**Comment by estelle** [comment]
Thanks @wbamberg. I merged your suggestion.

File: files/en-us/web/html/element/tbody/index.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ The following attributes are deprecated and should not be used. They are documen
 ## Usage notes
 
 - The `<tbody>` is placed after any {{HTMLElement("caption")}}, {{HTMLElement("colgroup")}}, and {{HTMLElement("thead")}} elements.
-- If {{HTMLElement("tr")}} elements are specified outside an existing `<tbody>` element as direct children of the {{HTMLElement("table")}}, these elements will be encapsulated by a separate `<tbody>` element generated by the browser.
+- If {{HTMLElement("tr")}} elements are specified as direct children of the {{HTMLElement("table")}} (see "tag omission" in the [technical summary](#technical_summary) for a description of when this is valid), then the markup generated by the browser will include a `<tbody>` element that encapsulates them. As a result, CSS selectors such as `table > tr` will not select these elements. See also the [[Not specifying a body](#not_specifying_a_body) example.
```
Comment (wbamberg) [inline files/en-us/web/html/element/tbody/index.md:45]:
```suggestion
- If {{HTMLElement("tr")}} elements are specified as direct children of the {{HTMLElement("table")}} (see "tag omission" in the [technical summary](#technical_summary) for a description of when this is valid), then the markup generated by the browser will include a `<tbody>` element that encapsulates them. As a result, CSS selectors such as `table > tr` will not select these elements. See also the [Not specifying a body](#not_specifying_a_body) example.
```

**Review by wbamberg — APPROVED** [review]
👍 thank you @estelle !

**Pull request merged**